frontend/src/features/admin-form/responses/ResponsesPage/storage/utils/downloadCsv.ts

```diff
@@ -0,0 +1,8 @@
+import FileSaver from 'file-saver'
+
+export const downloadResponseAttachment = async (
```

Comment:
used in a later PR? but seems like this can be inlined, doesn't really provide any abstraction

Reply:
used in later PR, but yea maybe... but also enforces a specific filename. will keep it as its own utli